data/names/ru.json

```diff
@@ -5870,7 +5870,624 @@
       "Янсен",
       "Янси",
       "Ян",
-      "Ярбро"
+      "Ярбро",
+      [ "Авель", "Abel" ],
+      [ "Авраам", "Abraham" ],
+      [ "Акерс", "Akers" ],
+      [ "Аланиз", "Alaniz" ],
+      [ "Александр", "Alexander" ],
+      [ "Алли", "Alley" ],
+      [ "Диамант", "Diamond" ],
+      [ "Амайя", "Amaya" ],
+      [ "Амарал", "Amaral" ],
+      [ "Амос", "Amos" ],
+      [ "Андрей", "Andrew" ],
+      [ "Арии", "Arias" ],
+      [ "Аруда", "Arruda" ],
+      [ "Арфа", "Harp" ],
+      [ "Байнум", "Bynum" ],
+      [ "Бэкон", "Bacon" ],
+      [ "Бальдерас", "Balderas" ],
+      [ "Банкс", "Banks" ],
+      [ "Барретт", "Barrett" ],
+      [ "Барриентос", "Barrientos" ],
+      [ "Барфилд", "Barfield" ],
+      [ "Бас", "Bass" ],
+      [ "Бассетт", "Bassett" ],
+      [ "Баумана", "Bauman" ],
```

Comment:
This one is interesting because it's *almost* correct, and could fly in a certain context. I believe this particular spelling is in reference to the [Bauman school](https://en.wikipedia.org/wiki/Bauman_Moscow_State_Technical_University), which in Russian (when worded as "School of Bauman") would be "Школа Баумана".

Here, it should be `Бауман`.